code/modules/mob/new_player/new_player.dm

```diff
@@ -342,6 +343,15 @@
 					var/obj/machinery/announcement_system/announcer = pick(announcement_systems)
 					announcer.announce("ARRIVAL", character.real_name, rank, list()) //make the list empty to make it announce it in common
 
+/mob/new_player/proc/AddEmploymentContract(mob/living/carbon/human/employee)
+	//TODO:  figure out a way to exclude wizards/nukeops/demons from this.
+	sleep(30)

```

Comment:
To give Ragin Mages a chance to work right.  I'm not sure if it's still needed, I'll look into it.